### PR TITLE
STEM AD: Create 10203 submission details report #18484

### PR DESCRIPTION
**Pull request by zurbergram**
## Description of change

Creates sidekiq job that emails out a 10203 specific spool file report 15mins after report for all submissions goes out

## Original issue(s)
department-of-veterans-affairs/va.gov-team#18484

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]

File: app/mailers/spool10203_submissions_report_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'reports/uploader'
+
+class Spool10203SubmissionsReportMailer < ApplicationMailer
+  REPORT_TEXT = '10203 spool submissions report'
+  RECIPIENTS = %w[
```
Comment (LindseySaari) [inline app/mailers/spool10203_submissions_report_mailer.rb:7]: Would the recipients be better suited for the settings.yml?

Comment (zurbergram) [reply app/mailers/spool10203_submissions_report_mailer.rb:7]: I can move them there

 I was following what https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/mailers/spool_submissions_report_mailer.rb is doing

Comment (LindseySaari) [reply app/mailers/spool10203_submissions_report_mailer.rb:7]: Ok, the settings.yml file just seems more fitting, but i'll leave it up to you.

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**